static/app/views/stories/storyFile.tsx

```diff
@@ -79,6 +79,9 @@ const FlexColumn = styled('section')`
 
 const StoryArea = styled('div')`
   overflow: scroll;
+  display: flex;
+  flex-direction: column;
+  gap: ${space(4)};
```

Comment:
Added this spacing so things will look better with multiple stories on the same page